draft-bbs-signatures.md

```diff
@@ -746,24 +755,87 @@ Inputs:
 
 Parameters:
 
-- r (REQUIRED), non-negative integer. The prime order of the G_1 and G_2 groups,
-     defined by the ciphersuite.
+- r (REQUIRED), non-negative integer. The prime order of the G1 and G2 groups, defined by the ciphersuite.
+- expand_length (REQUIRED), non-negative integer. The number of bytes to read from the xof function, defined by the ciphersuite.
+- dst (REQUIRED), octet string. Domain separation tag.
```

Comment:
It seems odd to list cipher suite parameters as required, aren't they all?

Reply:
Agreed, however there is the `key_info` parameter that is `OPTIONAL`.  IMO this is the reason that every other parameter is `REQUIRED`. Agreed though that this is not ideal, but it is "necessary evil" IMO.